extensionsGallery.json

```diff
@@ -3358,7 +3358,64 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+				"extensionId": "72",
```

Comment:
Please also update the TotalCount metadata value at the bottom of this file

Reply:
Hi thank you for guidelines,

so only increment by 1 ?

Reply:
Yup! (note the values will be different for this in insiders vs stable galleries since not all extensions are published to stable yet)